.gitignore

```diff
@@ -26,6 +26,7 @@ velocity.log.*
 
 # Local go binary directory
 /go-bin/
+/local/
```

Comment:
Why do we need 2 different folders for go binaries, shouldn't they be the same?

Reply:
I am not sure if we still use "go-bin" … but we can always remove it in a new PR right away if we don't.